packages/clerk-js/src/utils/url.ts

```diff
@@ -319,6 +324,10 @@ export function hasBannedProtocol(val: string | URL) {
   return BANNED_URI_PROTOCOLS.some(bp => bp === protocol);
 }
 
+export const isClerkImage = (src?: string): boolean => {
+  return !!CLERK_IMAGE_URL_BASES.some(base => src?.includes(base));
```

Comment:
🙃 Why `src` param is optional? I think we should make it mandatory and avoid `?.` in the function body.